tools/deb/sidecar.env

```diff
@@ -0,0 +1,50 @@
+# Environment variables used to configure istio startup
```

Comment:
Same - equivalent config files ( yaml, etc) don't seem to have it. 
Not sure what are the requirements for non-source code.